### ISSUE TITLE
expected precision field of Projection to be a number not a string in VegaLite 3.3.0 schema

### ISSUE THREAD

**Issue by DougBurke** [vega/vega-lite#5190]
I was surprised to find the `precision` field of `Projection` has a type of `string` and not `number`:

https://github.com/vega/schema/blob/master/vega-lite/v3.3.0.json#L8971

```
        "precision": {
          "description": "Sets the threshold for the projection’s [adaptive resampling](http://bl.ocks.org/mbostock/3795544) to the specified value in pixels. This value corresponds to the [Douglas–Peucker distance](http://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm). If precision is not specified, returns the projection’s current resampling precision which defaults to `√0.5 ≅ 0.70710…`.",
          "type": "string"
},
```

**Comment by domoritz** [comment]
Looks like Vega expects strings: https://vega.github.io/vega/docs/projections/

**Comment by domoritz** [comment]
Well, could be a bug in Vega. Could you check?

**Comment by domoritz** [comment]
The Vega schema says it's number: https://github.com/vega/vega/blob/master/packages/vega-schema/src/projection.js#L20.

**Comment by DougBurke** [comment]
Sorry - was in a meeting so couldn't investigate further.

**Comment by domoritz** [comment]
No problem. Again, thanks for catching and reporting the issue! I fixed it in #5191.
